src/connector/postgres.rs

```diff
@@ -700,6 +700,24 @@ mod tests {
         assert!(matches!(err.kind(), ErrorKind::AuthenticationFailed { user } if user == "WRONG"));
     }
 
+    #[tokio::test]
```

Comment:
Could this be actually tested in the integration tests for all (or most) databases?

Reply:
Meaning, in here https://github.com/prisma/quaint/blob/master/src/tests/query/error.rs